WordPress/gutenberg
# SelectControl: Fix labelPosition

<img width="307" alt="Screenshot of SelectControl labelPosition on top" src="https://user-images.githubusercontent.com/2322354/93507118-30ec0a80-f8eb-11ea-8ad3-b60a1b09ac7a.png">

This update fixes the default `labelPosition` for the `SelectControl` (defaulting to `top`). The `labelPosition` component API is the same as that from `InputControl`.

README documentation has been updated for the various controls to indicate `labelPosition`.

Thanks to @jorgefilipecosta for spotting this one!

## How has this been tested?

Tested locally in Storybook

* Run `npm run storybook:dev`
* Visit the `SelectControl` story (http://localhost:50240/?path=/story/components-selectcontrol--default)
* Change the `labelPosition` knob (below)
* Ensure the UI renders correctly

## Checklist:
- [n/a] My code is tested.
- [n/a] My code follows the accessibility standards. 
- [n/a] My code has proper inline documentation. 
- [n/a] I've updated all React Native files affected by any refactorings/renamings in this PR.
